silverbackis/common-js-bundle/1.0/post-install.txt

```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                    </>
+<bg=blue;fg=white>    What's next?    </>
+<bg=blue;fg=white>                    </>
+
+Now edit your .env file and set the IDs for scripts you want to enable
+
+* <fg=blue>Read</> the documentation at <comment>https://github.com/silverbackis/common-js-bundle</> to find out the blocks available and how to use the bundle.
```

Comment:
Lets not use the colors.

Reply:
Sure, no problem - the example in docs used colours just like this but I've removed them.